bucket/audacity242.json

```
@@ -0,0 +1,8 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",

```

```suggestion
    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
    "homepage": "https://www.audacityteam.org",
    "license": "GPL-2.0-or-later",
```
Add the homepage and license to pass checks

---

I added the license, it might have conflicts.